Doc 448 Windows | Containers (#455)
minio/docs: source/quickstart/container.rst

.. default-domain:: minio

This page deploys a standalone MinIO deployment as a as a container with an attached persistent volume for use with early development and evaluation environments.

For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.

Prerequisites
-------------

- `Podman <https://podman.io/getting-started/installation.html>`_ or `Docker <https://docs.docker.com/get-docker/>`_ installed
- Read, write, and delete access to the folder or drive used for the persistent volume

> Procedure

Reviewer: Lets not lose the connectivity step. For each H2 we can still do numbered steps for:

1. Start the container
2. Connect your Browser
3. (Optional) Connect mc

Strictly speaking you can run `mc` with a container but it does not work well in my experience. We'll have to walk them through downloading and running mc

---------

Use the following commands to create the container depending on the container service and local operating system.

We provide GNU/Linux, MacOS, or Windows instructions for the following container types:

- :ref:`quickstart-podman`
- :ref:`quickstart-docker-root`
- :ref:`quickstart-docker-regular-user`

.. _quickstart-podman:

> Podman

Author: Adding a note that the steps apply to both rootfull and rootless containers.

~~~~~~

.. tab-set::

   .. tab-item:: GNU/Linux or MacOS

      .. code-block:: shell
         :class: copyable

         podman run \
            -p 9000:9000 \
            -p 9001:9001 \
            -v ~/minio/data:/data \
            quay.io/minio/minio server /data --console-address ":9001"

> 

Reviewer: Also ensure we remind users that the command above remains attached to the terminal. On exit the process stops. I think for this pass we dont' need to go further.

   .. tab-item:: Windows

      .. code-block:: shell
         :class: copyable

         podman run \
            -p 9000:90900 \
            -p 9001:9001 \

>             -v D:\data:/data \

Author: `D:\` isn't the normal drive on Windows, `C:\` is.
I think it's better to specify a full path here.
I added a note that they can use any path to which the user has access.

            quay.io/minio/minio server /data --console-address ":9001"

.. _quickstart-docker-root:

Docker (as ``root`` user)
~~~~~~~~~~~~~~~~~~~~~~~~~

.. tab-set::

   .. tab-item:: GNU/Linux or MacOS

      .. code-block:: shell
         :class: copyable

         mkdir -p ~/minio/data

         docker run \
            -p 9000:9000 \
            -p 9001:9001 \

>             --name minio1 \

Reviewer: we could probably call it `minio` - unless that for some reason conflicts with the default pod name?

I'm also unsure the name matters if we aren't running as detatched

> .. default-domain:: minio

Reviewer: This is a heck of a job, but we're also playing with a *lot* of tabs and sections here.

We had I think originally talked about possibly breaking up the Containers section, and I'm starting to come around to that.

Maybe it makes sense to have `quickstart/docker.rst` and `quickstart/podman.rst` - we're duplicating some instructions, but the *flow* will be easier for users. 

Let me know what you think.

Author: AB was interested in mimicking the Download page on the website.
It has a `Docker` tab that just uses `podman` (no Docker instructions).

I suggest we leave it is a single page.
Maybe reconsider including Docker as an option?

Reviewer: I thought i commented but github says no.

Docker still has the lions share of the market and mind, so we have to address it alongside Podman for the forseeable future.

- Read, write, and delete access to the folder or drive used for the persistent volume

Procedure
---------
   
#. Start the container
   
   Select a container type to view instructions to create the container.
   Instructions are available for either GNU/Linux and MacOS or for Windows.

    .. important::
   
         The process is attached to the terminal session.
         Exiting the terminal ends the process.
   
   .. dropdown:: Podman (rootfull or rootless)
      :name: podman-root-rootless
   
      These steps work for both rootfull and rootless containers.

      .. tab-set::
   
         .. tab-item:: GNU/Linux or MacOS
   
            .. code-block:: shell
               :class: copyable

               podman run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  -v ~/minio/data:/data \
                  quay.io/minio/minio server /data --console-address ":9001"
   
            The example above works this way:
   
            - ``podman run`` starts the container.
            - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'

>             - When MinIO writes to ``/data`` within the container, the data actually writes to the ``~/minio/data`` local file path.

Author: How about:
```
When MinIO writes data to ``/data``, that data mirrors to the local path ``~/minio/data``, allowing it to persist between container restarts.
```

   
            .. code-block:: shell
               :class: copyable
   
               mkdir -p ~/minio/data
   
               docker run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  --name minio \
                  -v ~/minio/data:/data \
                  -e "MINIO_ROOT_USER=AKIAIOSFODNN7EXAMPLE" \
                  -e "MINIO_ROOT_PASSWORD=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" \
                  quay.io/minio/minio server /data --console-address ":9001"
         
            The example above works this way:
   
            - The ``mkdir`` command creates a new local directory at ``~/minio/data`` in your home directory.
            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``~/minio/data``) to the specified virtual container directory (``/data``).
            - When MinIO writes data to ``/data``, that data actually writes to the local path ``~/minio/data`` where it can persist between container restarts.
              You can replace ``~/minio/data`` with another local file location to which the user has read, write, and delete access.

>                  

Reviewer: If we keep the `-e` lines we should let users know this is setting envvars corresponding to :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`

                 
         .. tab-item:: Windows
   
            .. note::

               Make sure your AD/Windows user has write permissions to ``D:\data`` prior to using ``credentialspec=``
   
            .. code-block:: shell
               :class: copyable
   
               docker run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  --name minio1 \
                  --security-opt "credentialspec=file://myuser.json" \
                  -e "MINIO_ROOT_USER=AKIAIOSFODNN7EXAMPLE" \
                  -e "MINIO_ROOT_PASSWORD=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" \
                  -v D:\data:/data \
                  quay.io/minio/minio server /data --console-address ":9001"
   
            The example above works this way:
   
            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``D:\data``) to the specified virtual container directory (``/data``).
            - When MinIO writes data to ``/data``, that data actually writes to the local path ``D:\data`` where it can persist between container restarts.

>               You can replace ``D:\data`` with another local file location to which the user has read, write, and delete access.

Author: Removed

Procedure
---------
   
#. Start the container
   
   Select a container type to view instructions to create the container.
   Instructions are available for either GNU/Linux and MacOS or for Windows.

   .. dropdown:: Podman (rootfull or rootless)
      :name: podman-root-rootless
   
      These steps work for both rootfull and rootless containers.

      .. tab-set::
   
         .. tab-item:: GNU/Linux or MacOS
   
            .. code-block:: shell
               :class: copyable

               podman run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  -v ~/minio/data:/data \
                  -e "MINIO_ROOT_USER=ROOTNAME`" \
                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
                  quay.io/minio/minio server /data --console-address ":9001"
   
            The example above works this way:
   
            - ``podman run`` starts the container.
              The process is attached to the terminal session and ends when exiting the terminal.
            - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'

>             - ``-e`` sets the environment variables ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``, respectively.

Reviewer: ```suggestion
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively. These set the :ref:`root user credentials <minio-users-root>`
```

              Change the example values to use for your container.
            - When MinIO writes data to ``/data``, that data mirrors to the local path ``~/minio/data``, allowing it to persist between container restarts.

>               You can set any file path to which the user has read, write, and delete permissions to use.

Reviewer: Move to line 52

   
         .. tab-item:: Windows
   
            .. code-block:: shell
               :class: copyable
   
               podman run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  -v D:\data:/data \
                  -e "MINIO_ROOT_USER=ROOTNAME`" \
                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
                  quay.io/minio/minio server /data --console-address ":9001"
   
            The example above works this way:
   
            - ``podman run`` starts the container.
            - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'

>             - ``-e`` sets the environment variables ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``, respectively.

Reviewer: ```suggestion
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively. These set the :ref:`root user credentials <minio-users-root>`
```

              Change the example values to use for your container.

>             - When MinIO writes data to ``/data``, that data mirrors to the local path ``~/minio/data``, allowing it to persist between container restarts.

Reviewer: ```suggestion
            - When MinIO writes data to ``/data``, that data mirrors to the local path ``D:\data``, allowing it to persist between container restarts.
```

Author: nice catch

>               You can set any file path to which the user has read, write, and delete permissions to use.

Reviewer: Move this to line 75

>    .. dropdown:: Docker as ``root``

Reviewer: Docker (Rootfull) - for parity with others?

      :name: docker-rootfull
   
      .. tab-set::
   
         .. tab-item:: GNU/Linux or MacOS
   
            .. code-block:: shell
               :class: copyable
   
               mkdir -p ~/minio/data
   
               docker run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  --name minio \
                  -v ~/minio/data:/data \
                  -e "MINIO_ROOT_USER=ROOTNAME`" \
                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
                  quay.io/minio/minio server /data --console-address ":9001"
         
            The example above works this way:
   
            - The ``mkdir`` command creates a new local directory at ``~/minio/data`` in your home directory.
            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``~/minio/data``) to the specified virtual container directory (``/data``).
            - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'

>             - ``-e`` sets the environment variables ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``, respectively.

Reviewer: ```suggestion
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively. These set the :ref:`root user credentials <minio-users-root>`
```

              Change the example values to use for your container.
            - When MinIO writes data to ``/data``, that data mirrors to the local path ``~/minio/data``, allowing it to persist between container restarts.

>               You can replace ``~/minio/data`` with another local file location to which the user has read, write, and delete access.

Reviewer: Move to 107

                 
         .. tab-item:: Windows
   
            .. code-block:: shell
               :class: copyable
   
               docker run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  --name minio1 \
                  -v D:\data:/data \
                  -e "MINIO_ROOT_USER=ROOTUSER" \
                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
                  quay.io/minio/minio server /data --console-address ":9001"
               
            The example above works this way:
   
            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``D:\data``) to the specified virtual container directory (``/data``).
            - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'

>             - ``-e`` sets the environment variables ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``, respectively.

Reviewer: ```suggestion
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively. These set the :ref:`root user credentials <minio-users-root>`
```

              Change the example values to use for your container.
            - When MinIO writes data to ``/data``, that data mirrors to the local path ``D:\data``, allowing it to persist between container restarts.

>               You can replace ``D:\data`` with another local file location to which the user has read, write, and delete access.

Reviewer: Move to 131

            
   .. dropdown:: Docker (Rootless)
      :name: docker-rootless
   
      .. tab-set::
   
         .. tab-item:: GNU/Linux or MacOS
   
            .. code-block:: shell
               :class: copyable
   
               mkdir -p ${HOME}/data
   
               docker run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  --user $(id -u):$(id -g) \
                  --name minio1 \
                  -e "MINIO_ROOT_USER=ROOTUSER" \
                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
                  -v ${HOME}/minio/data:/data \
                  quay.io/minio/minio server /data --console-address ":9001"
         
            The example above works this way:
   
            - The ``mkdir`` command creates a new local directory at ``~/minio/data`` in your home directory.
            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``~/minio/data``) to the specified virtual container directory (``/data``).
            - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'

>             - ``-e`` sets the environment variables ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``, respectively.

Reviewer: ```suggestion
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively. These set the :ref:`root user credentials <minio-users-root>`
```

              Change the example values to use for your container.
            - When MinIO writes data to ``/data``, that data actually writes to the local path ``~/minio/data`` where it can persist between container restarts.

>               You can replace ``${HOME}/minio/data`` with another location in the user's home directory to which the user has read, write, and delete access.

Reviewer: Move to 164

                 
         .. tab-item:: Windows
   
            .. code-block:: shell
               :class: copyable
   
               docker run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  --name minio1 \
                  -e "MINIO_ROOT_USER=ROOTUSER" \
                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
                  -v D:\data:/data \
                  quay.io/minio/minio server /data --console-address ":9001"
   
            The example above works this way:
   
            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``D:\data``) to the specified virtual container directory (``/data``).
             - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'

>             - ``-e`` sets the environment variables ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``, respectively.

Reviewer: ```suggestion
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively. These set the :ref:`root user credentials <minio-users-root>`
```

              Change the example values to use for your container.
           - When MinIO writes data to ``/data``, that data actually writes to the local path ``D:\data`` where it can persist between container restarts.

>               You can replace ``D:\data`` with another local file location to which the user has read, write, and delete access.

Reviewer: Move to 188

#. Connect your browser

   Open http://127.0.0.1:9000 in a web browser to access the :ref:`MinIO Console <minio-console>`. 
   Alternatively, you can enter any of the network addresses specified as part of the container command output.

   Access the :ref:`minio-console` by going to a browser and enter one of the Console address specified in the container command's output.
   For example, :guilabel:`Console: http://192.0.2.10:9001 http://127.0.0.1:9001` in the example output indicates two possible addresses to use for connecting to the Console.

>    If you go to port ``9000``, which is used for connecting to the API, you are automatically redirected to the Console port.

Reviewer: ```suggestion
   If you go to port ``9000``, which is used for connecting to the API, MinIO automatically redirects to the Console port.
```

Removing passive

Reviewer: Also maybe move this up as part of 196 and simplify as "MinIO automatically redirects browser access from the API port (``9000``) to the Console port.

Author: Moved up.

>    Log in to the Console with the credentials you defined in the ``-e`` environment variables you defined for the container for ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``.

Reviewer: Double usage of 'defined'. Spitballing:

```suggestion
   Log in to the Console with the credentials you defined in the ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD`` environment variables.
```

>       In your browser, go to http://127.0.0.1:9000.

Reviewer: Repeats above - cruft to remove?

Author: Yep, deleted.

   You can use the MinIO Console for general administration tasks like Identity and Access Management, Metrics and Log Monitoring, or Server Configuration. 
   Each MinIO server includes its own embedded MinIO Console.

   .. image:: /images/minio-console-login.png
      :width: 400px
      :alt: MinIO Console displaying login screen
      :align: center

>    From the console, you can create and manage buckets, add objects, and many other tasks.

Reviewer: ```suggestion
   From the Console, you can create and manage buckets, add objects, and many other tasks.
```

Reviewer: Actually we already state you can use the Console for all kinds of tasks, so we can just drop this

Author: Agree. Moved the login image to right under the Log in line.
Deleted 215.
Moved the the "for more information" to under the bucket image.
Did the same for the MacOS and Windows pages.

   For more information, see the :ref:`minio-console` documentation.

   .. image:: /images/minio-console-buckets.png
      :width: 600px
      :alt: MinIO Console displaying bucket start screen
      :align: center

> #. `(Optional)` Connect mc

Reviewer: Connect using the MinIO Client Command Line Interface

More verbose, but also clearer as to what we are asking them to do. If accepted I'll make this change in the Linux PR.

Author: Changed to `Install the MinIO Client` to match the other pages.

                     curl -O https://dl.min.io/client/mc/release/darwin-amd64/mc
                     chmod +x mc
                     sudo mv mc /usr/local/bin/mc

         .. tab-item:: Homebrew
      
            Run the following commands to install the latest stable MinIO Client package using `Homebrew <https://brew.sh>`_.
      
            .. code-block:: shell
               :class: copyable
      
               brew install minio/stable/mc
      
      Use :mc-cmd:`mc alias set` to quickly authenticate and connect to the MinIO deployment.
      
      .. code-block:: shell
         :class: copyable
      
         mc alias set local http://127.0.0.1:9000 {MINIO_ROOT_USER} {MINIO_ROOT_PASSWORD}
         mc admin info local
      
      Replace ``{MINIO_ROOT_USER}`` and ``{MINIO_ROOT_PASSWORD}`` with the credentials you defined for the container with the ``-e`` flags.

      The :mc-cmd:`mc alias set` takes four arguments:
   
      - The name of the alias
      - The hostname or IP address and port of the MinIO server
      - The Access Key for a MinIO :ref:`user <minio-users>`
      - The Secret Key for a MinIO :ref:`user <minio-users>`

>    For additional details about this command, see :ref:`alias`.

Reviewer: Indent one more

      
   .. dropdown:: Windows
   
      Download the standalone MinIO server for Windows from the following link:
   
      https://dl.min.io/client/mc/release/windows-amd64/mc.exe
   
      Double click on the file to run it.
      Or, run the following in the Command Prompt or PowerShell.
      
      .. code-block::
         :class: copyable
   
         \path\to\mc.exe --help
         
      Use :mc-cmd:`mc alias set` to quickly authenticate and connect to the MinIO deployment.
   
      .. code-block:: shell
         :class: copyable
   

>          mc alias set local http://127.0.0.1:9000 {MINIO_ROOT_USER} {MINIO_ROOT_PASSWORD}

Reviewer: mc -> mc.exe

              You can set any file path to which the user has read, write, and delete permissions to use.
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively.
              These set the :ref:`root user credentials <minio-users-root>`.
              Change the example values to use for your container.
   
   .. dropdown:: Docker as ``rootfull``
      :name: docker-rootfull
   
      .. tab-set::
   
         .. tab-item:: GNU/Linux or MacOS
   
            .. code-block:: shell
               :class: copyable
   
               mkdir -p ~/minio/data
   
               docker run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  --name minio \
                  -v ~/minio/data:/data \
                  -e "MINIO_ROOT_USER=ROOTNAME`" \
                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
                  quay.io/minio/minio server /data --console-address ":9001"
         
            The example above works this way:
   
            - The ``mkdir`` command creates a new local directory at ``~/minio/data`` in your home directory.

>             - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``~/minio/data``) to the specified virtual container directory (``/data``).

Author: This is good.
I love having another pair of eyes on what I write.

            - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'
              When MinIO writes data to ``/data``, that data mirrors to the local path ``D:\data``, allowing it to persist between container restarts.
              You can replace ``D:\data`` with another local file location to which the user has read, write, and delete access.
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively.
              These set the :ref:`root user credentials <minio-users-root>`.
              Change the example values to use for your container.
            
   .. dropdown:: Docker (Rootless)
      :name: docker-rootless
   
      .. tab-set::
   
         .. tab-item:: GNU/Linux or MacOS
   
            .. code-block:: shell
               :class: copyable
   
               mkdir -p ${HOME}/data
   
               docker run \
                  -p 9000:9000 \
                  -p 9001:9001 \

>                   --user $(id -u):$(id -g) \

Reviewer: IMO we should call this out in the bullet list as to why it's necessary (I'm guessing it just tells Docker to run as the current user/group?)

            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``~/minio/data``) to the specified virtual container directory (``/data``).
            - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'
              When MinIO writes data to ``/data``, that data actually writes to the local path ``~/minio/data`` where it can persist between container restarts.
              You can replace ``${HOME}/minio/data`` with another location in the user's home directory to which the user has read, write, and delete access.
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively.
              These set the :ref:`root user credentials <minio-users-root>`.
              Change the example values to use for your container.
                 
         .. tab-item:: Windows
   
            .. code-block:: shell
               :class: copyable
   
               docker run \
                  -p 9000:9000 \
                  -p 9001:9001 \
                  --name minio1 \

>                   -e "MINIO_ROOT_USER=ROOTUSER" \

Reviewer: Just checking --user isn't needed here.

Author: Circling back.
Requires a Windows group Managed Service Account and the `security-opt credentialspec=""` flag/option.
Added the flag, links to Microsoft docs, and the bullet point about the --secruity-opt .

                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
                  -v D:\data:/data \
                  quay.io/minio/minio server /data --console-address ":9001"
   
            The example above works this way:
   
            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``D:\data``) to the specified virtual container directory (``/data``).
            - ``-p`` binds a local port to a container port.
            - ``-v`` sets a file path as a persistent volume location for the container to use.'
              When MinIO writes data to ``/data``, that data actually writes to the local path ``D:\data`` where it can persist between container restarts.
              You can replace ``D:\data`` with another local file location to which the user has read, write, and delete access.
            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively.
              These set the :ref:`root user credentials <minio-users-root>`.
              Change the example values to use for your container.

> #. Connect your browser

Reviewer: ```suggestion
#. Connect your Briowser to the MinIO Server
```

Author: Fixed (with spelling right!) on Container, MacOS, and Windows doc

   Open http://127.0.0.1:9000 in a web browser to access the :ref:`MinIO Console <minio-console>`. 
   MinIO automatically redirects browser access from the API port (``9000``) to the Console port.
   Alternatively, you can enter any of the network addresses specified as part of the container command output.

   Access the :ref:`minio-console` by going to a browser and enter one of the Console address specified in the container command's output.

>    For example, :guilabel:`Console: http://192.0.2.10:9001 http://127.0.0.1:9001` in the example output indicates two possible addresses to use for connecting to the Console.

Reviewer: This feels duplicated (github might have dropped a prior comment on this)

204 and 206 are saying the same thing, but 206 goes into the correct detail. Can we combine?
